packages/ens-utils/tsup.config.ts

```diff
@@ -2,9 +2,10 @@ import { defineConfig } from "tsup";
 
 export default defineConfig({
   entry: ["src/index.ts"],
-  splitting: true,
-  sourcemap: false,
+  splitting: false,
+  sourcemap: true,
   clean: true,
+  bundle: true,
   dts: true,
   treeshake: true,
```

Comment:
I notice how we have `treeshake` configured for `ens-utils` but not other packages. Is there any special reason all packages don't use the same config?